### ISSUE TITLE
Context manager methods no longer work

### ISSUE THREAD

**Issue by vberlier** [microsoft/pylance-release#717]
Pylance `2020.12.1` throws an internal error with the following code:

```python
from contextlib import contextmanager

class Hello:
    @contextmanager
    def foo(self):
        yield

with Hello().foo():
    pass
```

```
TypeError: Cannot read property 'category' of undefined
    at Object.isUnknown (pyright-internal/src/analyzer/types.ts:1486:17)
    at isPartlyUnknown (pyright-internal/src/analyzer/typeUtils.ts:1362:9)
    at Object.isPartlyUnknown (pyright-internal/src/analyzer/typeUtils.ts:1410:21)
    at reportPossibleUnknownAssignment (pyright-internal/src/analyzer/typeEvaluator.ts:8712:20)
    at getTypeFromMemberAccessWithBaseType (pyright-internal/src/analyzer/typeEvaluator.ts:3816:13)
    at getTypeFromMemberAccess (pyright-internal/src/analyzer/typeEvaluator.ts:3551:34)
    at getTypeOfExpression (pyright-internal/src/analyzer/typeEvaluator.ts:815:30)
    at getTypeFromCall (pyright-internal/src/analyzer/typeEvaluator.ts:5004:32)
    at getTypeOfExpression (pyright-internal/src/analyzer/typeEvaluator.ts:825:30)
    at Vt (pyright-internal/src/analyzer/typeEvaluator.ts:11442:24)
```

**Comment by erictraut** [comment]
Thanks for the bug report. This is caused by a recent regression that slipped through our tests. It will be fixed in the next release.

**Comment by jakebailey** [comment]
This issue has been fixed in version 2020.12.2, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/master/CHANGELOG.md#2020122-11-december-2020
